src/components/Icons/EventsModalIcons.jsx

```diff
@@ -38,4 +38,12 @@ const FileUploadIcon = createIcon({
   ),
 });
 
-export { FileUploadIcon, CreateEventIcon, CancelIcon };
+const theme = extendTheme({
+  icons: {
+    FileUploadIcon,
+    CreateEventIcon,
+    CancelIcon,
+  },
+});
+
+export { theme, FileUploadIcon, CreateEventIcon, CancelIcon };
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.